skl2onnx/operator_converters/TextVectorizer.py

```diff
@@ -8,6 +8,51 @@
 from ..common._registration import register_converter
 
 
+def _intelligent_split(text, op, tokenizer, existing):
+    """
+    Splits text into tokens. *scikit-learn*
+    merges tokens with ``' '.join(tokens)``
+    to name ngrams. ``'a  b'`` could be ``('a ', 'b')``
+    or ``('a', ' b')``.
+    See `ngram sequence
+    <https://github.com/scikit-learn/scikit-learn/blob/master/
+    sklearn/feature_extraction/text.py#L169>`_.
+    """
+    if op.analyzer == 'word':
+        if op.ngram_range[0] == op.ngram_range[1] == 1:
+            spl = [text]
+        elif '  ' not in text:
+            if text.startswith(' '):
```

Comment:
Line 25 to line 33 could just be replaced with:
spl = text.strip().split()

Reply:
Not exactly because I need to remove only the first space. If there are two spaces at the beginning, I'll only remove one.

Reply:
But then you have a check for two spaces, which would fail and this wouldn't be executed in case there are two spaces.  
elif '  ' not in text:

Reply:
You are right, I also need to modify the way it is split: only one space should be used.

Reply:
In fact, I changed the way I do. This code is only used when space can be part of tokens. There are some ambiguities we can clear but not all of them.